nestor_api/config/kubernetes.py

```diff
@@ -0,0 +1,12 @@
+"""Nestor-api kubernetes configuration"""
+import os
+
+
+# pylint: disable=too-few-public-methods
+class KubernetesConfiguration:
+    """Kubernetes configuration"""
+
+    @staticmethod
+    def get_http_proxy() -> str:
+        """Returns the kubernetes http proxy"""
+        return os.environ["NESTOR_K8S_HTTP_PROXY"]
```

Comment:
Also the file name.

I think we should be consistent between k8s and kubernetes, here the variables has `K8S`, builders are in a `k8s` folder, we could change to `kubernetes` but just to have same naming would be easier I think

```suggestion
"""Nestor-api k8s configuration"""
import os


# pylint: disable=too-few-public-methods
class K8sConfiguration:
    """K8s configuration"""

    @staticmethod
    def get_http_proxy() -> str:
        """Returns the k8s http proxy"""
        return os.environ["NESTOR_K8S_HTTP_PROXY"]
```

Reply:
I will go with `k8s` then